backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/service/ShopService.java

```diff
@@ -0,0 +1,18 @@
+package com.google.gpay.virtualqueue.backendservice.service;
+
+import com.google.gpay.virtualqueue.backendservice.model.Shop;
+import com.google.gpay.virtualqueue.backendservice.respository.InMemoryShopRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ShopService {
+
+	@Autowired
+	InMemoryShopRepository shopRepository;
```

Comment:
Wondering if we should be specifying the interface here?
Read up on dependency injection.

Reply:
Yeah I guess that if we make a call to the interface we can abstract the actual implementation. I will make that change.